scripts/lib-franklin.js

```diff
@@ -615,6 +615,13 @@ export function decorateButtons(element) {
           a.nextSibling.remove();
           return;
         }
+        if (up.childNodes.length === 1 && up.tagName === 'P' && up.innerText.startsWith('?')) {
```

Comment:
could you add a comment like `// Example: <p><a href="example.com">? Text</a></p>` please?